Pyrado/pyrado/sampling/parallel_rollout_sampler.py

```diff
@@ -48,6 +48,9 @@
 from pyrado.spaces.box import InfBoxSpace
 
 
+NO_SEED_PASSED = object()
```

Comment:
just out of curiosity: why `object()`?

Reply:
It creates a fresh instance such that the below `is` statement ist reliable. Could be anything else, e.g. an empty dict, too. But imho using a plain object instance is less confusing.